packages/components/src/components/post-avatar-picture/post-avatar-picture.scss

```diff
@@ -0,0 +1,35 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+:host {
+  display: block;
```

Comment:
This would lead to a gap below of the picture.
![image](https://github.com/user-attachments/assets/4022571d-0104-481a-ad71-58f971a292d8)
![image](https://github.com/user-attachments/assets/010f783c-51db-46fa-948d-3e9f613d902d)

Why you think this needs to be inline-block?

Reply:
HTML `<img>` are inline elements so I would expect the avatar picture to be too.
Where is this gap coming from?

Reply:
Use inline-block and also replace `aspect-ratio` with a fix width/height.